extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -410,4 +452,20 @@ private function formatTotalEditorsNumber( $editors ) {
 
 		return $editors;
 	}
+
+	/**
+	 * @param $userId
+	 * @return string name of DS svg file with proper badge
+	 */
+	private function getUserBadge( $userId ) {
```

Comment:
I am not sure if that's an appropriate place. The application should depend on the DS, not the other way round. A more adequate place would be (from my perspective) AvatarService